README.md

```diff
@@ -17,7 +17,8 @@
 [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/nlohmann/json.svg)](http://isitmaintained.com/project/nlohmann/json "Average time to resolve an issue")
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/289/badge)](https://bestpractices.coreinfrastructure.org/projects/289)
 [![GitHub Sponsors](https://img.shields.io/badge/GitHub-Sponsors-ff69b4)](https://github.com/sponsors/nlohmann)
-
+[![Conan](https://api.bintray.com/packages/conan/conan-center/nlohmann_json%3A_/images/download.svg) ](https://bintray.com/conan/conan-center/nlohmann_json%3A_/_latestVersion)
```

Comment:
I find this badge a bit confusing, because it does not mention Conan but just "download" - and the right place to download the library would be https://github.com/nlohmann/json/releases for which we already have a badge.

Reply:
Agree, dont see though a possibility of changing this, therefore dropping it

Maybe @uilianries can change the badge

Reply:
Hi! Can't change it because is related to Bintray feature. But, we can use a custom shield:

[![Conan](https://img.shields.io/badge/Conan-Bintray-blue)](https://bintray.com/conan/conan-center/nlohmann_json%3A_/_latestVersion)